webrender/src/picture.rs

```diff
@@ -596,9 +596,18 @@ impl PicturePrimitive {
 
     pub fn write_gpu_blocks(&self, request: &mut GpuDataRequest) {
         match self.kind {
-            PictureKind::TextShadow { .. } |
-            PictureKind::Image { .. } => {
-                request.push([0.0; 4]);
+            PictureKind::TextShadow { .. } => {
```

Comment:
There are some code paths that rely on knowing the number of GPU blocks that a primitive writes (the brush / segment code). In this case, it's actually fine, since ```PictureKind::Image``` never passes through that path. However, it may do in the future. Could you add a comment here noting that this code will need to change if/when we start pushing Image pictures through the brush path?